frameworks/Scala/zio-http/src/main/scala/Main.scala

```diff
@@ -0,0 +1,41 @@
+import zhttp.http._
+import zhttp.service.Server
+import zio.{App, ExitCode, URIO}
+import io.netty.handler.codec.http.{HttpHeaderNames => JHttpHeaderNames, HttpHeaderValues => JHttpHeaderValues}
+
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+object WebApp extends App {
+
+  val message: String = "Hello, World!"
+def createDate: String = DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now)
```

Comment:
To stay in the idiomatic realm of using ZIO-related libraries this should be suspended in ZIO (`IO.effectTotal` on `ZonedDateTime.now`, or `zio.clock.currentDateTime` etc.)

Reply:
We will move certain "basic" headers to the core. For eg: `Date` and `Server`. Almost all libraries do it this way for performance reasons. Also, the code above will look a lot more concise.